### DISCUSSION TITLE
How can  I render image manually image in avatar ?

### DISCUSSION THREAD

**Discussion by roman-hrybinchuk** [chakra-ui/chakra-ui#3918] [General]
I want to use Next.js  image optimization feature. But I don't quite understand how can I put it in the avatar, since it uses the link not the component.

**Comment by ivan-kleshnin** [comment]
I guess for now the only options are:
- not use Next Image for Avatar (it's easy to replace Chakra's `Image` with `NextImage + chakra factor`. But `Avatar` is a different story) and live without those responsive features.
- make and support a fork of all Avatar-related component based on `NextImage` (or reimplement Chakra's `Avatar` functionality yourself).

😞 

If someone got the time to do the latter (should be an easy library for an open-source beginner) – please share it here 🙏
